doc/shell/shell_guide.md

```diff
@@ -0,0 +1,817 @@
+Fix: I am sure of this verication result.  
+Save: I am not sure of this verication result. I will decide it later.  
+
+Sometimes, we cannot find the failure reason from the screen output directly. At this time, we need login the test machine, go to the case path, check the logs. If we still cannot find the failure reason from the logs, we should run the case step by step on this machine to find the failure reason.  
+We should verify the crash failures first.  
+
+## 4.2 Verify code coverage test result
+Go to QA homepage and find the 'code coverage' node in the left area, click the link of the latest result.  
+![code coverage 1](./image19.png)
+Click 'shell' link.  
+![code coverage 2](./image10.png)  
+Check the value of 'Coverage' column. The coverage value should not lower than previous code coverage test result.  
+![code coverage 3](./image11.png) 
+
+## 4.3 Report issues
+### crash issue  
+If there is a crash, report it following the rules in this link: [‘How to Report Regression Crash Issues’](http://jira.cubrid.org/browse/CUBRIDQA-1?focusedCommentId=4739244&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4739244)  
+
+### normal issue  
+If the the test result is as expected, first we should confirm whether it is revise required. If not, report it as an issue on jira.   
+
+## 4.4 Execute a shell test by sending message
+We can use the regression tools to trigger a test.  
+1. Change the parameters in ~/CTP/conf/shell_template.conf on controller node.    
+2. Send a message to start the test  
+Login: message@192.168.1.91  
+Send test message as:  
+```
+sender.sh QUEUE_CUBRID_QA_SHELL_LINUX http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8330-d4d8464/drop/CUBRID-10.2.0.8330-d4d8464-Linux.x86_64.sh shell default
+```
+3. Start consumer on the controller node  
+If the consumer is not started, login the controller node to start it.  
+```
+cd ~
+sh start_test.sh 
+```
+4. Check test result  
+The results will be uploaded to qahome automatically. You can follow ['4.2 Verify dailyqa test results'](#4.2_Verify_dailyqa_test_results) to check the test results. 
+
+## 4.5 Execute a code coverage test
+For code coverage test, just need to send a message.  
+Login: message@192.168.1.91  
+Send test message as:  
+```
+cd ~/manual
+sh sender_code_coverage_testing_message.sh Queue:QUEUE_CUBRID_QA_SHELL_LINUX Build URL:http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8270-c897055/drop/CUBRID-10.2.0.8270-c897055-gcov-Linux.x86_64.tar.gz Source URL:http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8270-c897055/drop/cubrid-10.2.0.8270-c897055-gcov-src-Linux.x86_64.tar.gz Category:shell
+```
+The result will be uploaded to qahome automatically.  
+To check the result, please refer to ['Verify code coverage test result'](#Verify-code-coverage-test-result) 
+
+
+# 5 shell case standards
+## 5.1 case path standard
+### new feature path:  
+We created folders for each cubrid version like this:  

```

Comment:
Please use the description "How to make a shell cases" in 
http://jira.cubrid.org/secure/attachment/1013440/2019%20CUBRID%20Global%20QA_20190416_v4.pdf